Doc/library/os.rst

```diff
@@ -5164,6 +5164,10 @@ Miscellaneous System Information
 
    .. versionadded:: 3.4
 
+   .. versionchanged:: 3.13
+      if ``-X cpu_count=<n>`` is given, :func:`cpu_count` will return the overrided
+      value.
```

Comment:
Can you integrate this text in the doc above, and here just mention that the function takes -X cpu_count in account?

```suggestion
      if :samp:`-X cpu_count={n}` is given, :func:`cpu_count` will return the overrided
      value *n*.
```